app/assets/stylesheets/components/_card.scss

```diff
@@ -1,6 +1,6 @@
 .card {
   background-color: #fff;
-  box-shadow: 0px 1px 1px 0px rgba(0,0,0,0.05);
+  box-shadow: 0px 1px 1px 0px rgba(0,0,0,0.1);
```

Comment:
Commas in function arguments should be followed by a single space<br>`0px` should be written without units as `0`